RestoreTestKBFromProd/Program.cs

```diff
@@ -47,11 +67,28 @@ private async Task Process()
             else
             {
                 var publishedKbs = allKbs.Where(kb => !string.IsNullOrEmpty(kb.LastPublishedTimestamp)).ToList();
-
-                Console.WriteLine($"Total KB Count: {allKbs.Count}, published Kbs Count: {publishedKbs.Count}, unpublished Kbs (can't be restored): {allKbs.Count - publishedKbs.Count}");
-
                 if (publishedKbs.Count > 0)
                 {
+                    if (enableMultipleLanguages)
+                    {
+                        Console.WriteLine($"Total KB Count: {allKbs.Count}, published Kbs Count: {publishedKbs.Count}, unpublished Kbs: {allKbs.Count - publishedKbs.Count}");
```

Comment:
Why is this moved inside the if (), just curious?

Reply:
removed the "can't be restored" part